PWGHF/HFC/Utils/utilsCorrelations.h

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file utilsCorrelations.h
+/// \brief Utilities for HFC analyses
+
+#ifndef PWGHF_HFC_UTILS_UTILSCORRELATIONS_H_
+#define PWGHF_HFC_UTILS_UTILSCORRELATIONS_H_
+
+#include <TString.h>
+#include <cmath>
```

Comment:
```suggestion
#include <cmath>
#include <TString.h>
```
Include the header which defines `o2::constants::math::PI`.

Reply:
You also need to include `TPDGCode.h` for the named PDG codes.